src/blocks/BlockWithImage.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import castle from '../assets/image/login.png';
+import mask from '../assets/image/mask.png';
+
+export const BlockWithImage = () => {
+  return (
+    <section className="relative flex flex-col items-center justify-center w-full px-4 py-12 bg-gray-950 h-max md:p-20 xl:flex-row">
+      <div className="absolute top-0 w-full h-[160px] bg-gradient-to-b from-gray-800/100 via-red-800/10 to-transparent z-10"></div>
+      <div className="flex flex-col items-center justify-start xl:w-1/2">
+        <h2 className="mb-4 text-3xl font-bold text-white">
+          Odkryj, jak nasze narzędzie ułatwia zarządzanie postacią i kampanią.
+        </h2>
+        <p className="mb-4 text-base font-semibold text-white">
+          Nasze narzędzie sprawia, że tworzenie i zarządzanie postacią jest
+          proste i intuicyjne. Dzięki zaawansowanym funkcjom, możesz skupić się
+          na przygodzie, a nie na liczbach.
+        </p>
+      </div>
+      <div className="relative flex flex-col items-center justify-center min-h-[380px] h-[50vh] xl:w-1/2 bg-cover">
+        <img
+          src={castle}
+          alt="image castle"
```

Comment:
Trochę słaby alt, powinien on opisywać co się na obrazku znajduje tak aby użytkownik któremu się on nie załaduje, wiedział o co chodzi. To trochę ubogo moim zdaniem.